bedms/attr_standardizer.py

```diff
@@ -0,0 +1,208 @@
+import logging
+from typing import Dict, Tuple, Union
+
+import peppy
+import torch
+import torch.nn as nn
+import torch.nn.functional as torch_functional
+
+from .const import (
+    AVAILABLE_SCHEMAS,
+    CONFIDENCE_THRESHOLD,
+    DROPOUT_PROB,
+    EMBEDDING_SIZE,
+    HIDDEN_SIZE,
+    INPUT_SIZE_BOW_BEDBASE,
+    INPUT_SIZE_BOW_ENCODE,
+    INPUT_SIZE_BOW_FAIRTRACKS,
+    OUTPUT_SIZE_BEDBASE,
+    OUTPUT_SIZE_ENCODE,
+    OUTPUT_SIZE_FAIRTRACKS,
+    PROJECT_NAME,
+    SENTENCE_TRANSFORMER_MODEL,
+)
+from .model import BoWSTModel
+from .utils import (
+    data_encoding,
+    data_preprocessing,
+    fetch_from_pephub,
+    get_any_pep,
+    load_from_huggingface,
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(PROJECT_NAME)
+
+
+class AttrStandardizer:
+    def __init__(self, schema: str, confidence: int = CONFIDENCE_THRESHOLD) -> None:
+        """
+        Initializes the attribute standardizer with user provided schema, loads the model.
+
+        :param str schema: User provided schema, can be "ENCODE" or "FAIRTRACKS"
+        :param int confidence: Confidence threshold for the predictions.
+        """
+        self.schema = schema
+        self.model = self._load_model()
```

Comment:
Based on `self._load_model()` it doesnt seem like I can train my own? Is this possible? Probably a bigger discussion, but this seems tightly coupled to you training it "behind the scenes" and a user can't train their own

Reply:
Presently, the user cannot use `bedms` on a model they have trained. 
For that, we would have to first make the updated training scripts available to users. And also provide documentation on preprocessing their training scripts.
I agree that this needs further discussion though.